### PR TITLE
Add Arthurtilly to the CODEOWNERS file

### PR DESCRIPTION
**Pull request by Reonu**

### CODE REVIEW

**Review by arthurtilly — CHANGES_REQUESTED** [review]
dont trust this guy

**Comment by gheskett** [comment]
I'll come back to this later when I have time, but I feel like there's gotta be a way to make the CODEOWNERS not only look at the bottom entry.

File: CODEOWNERS [hunk 1]
```diff
@@ -5,6 +5,7 @@
 *       @Reonu
 *       @thecozies
 *       @gheskett
+*       @arthurtilly
```
Comment (gheskett) [inline CODEOWNERS:8]: Try this actually. I think the issue is that we're re-establishing a global CODEOWNER every time, but by adding them as part of the same line, it should allow everyone to share the role of a global CODEOWNER...I think.

```suggestion
*       @Reonu @thecozies @gheskett @arthurtilly
```

**Review by gheskett — CHANGES_REQUESTED** [review]

**Review by gheskett — APPROVED** [review]
This is now ready to go. (Please squash this one.)

**Comment by Reonu** [comment]
Closing since Arthur doesn't trust that guy

**Review by arthurtilly — APPROVED** [review]

**Pull request merged**